src/platform/common/process/logger.unit.test.ts

```diff
@@ -1,114 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-import { expect } from 'chai';
```

Comment:
Removed the tests, as they were pointless, basically this test was just testing logging